cli/flox-rust-sdk/src/models/env_registry.rs

```diff
@@ -134,6 +141,9 @@ pub struct RegistryEntry {
     /// The list of environments that have existed at this path
     /// since the last time environments were garbage collected.
     pub envs: Vec<RegisteredEnv>,
+    /// The PIDs of current activations
+    #[serde(default)]
```

Comment:
Suggestion: you may want to restrict the size of the `HashSet` that `proptest` generates so it doesn't unnecessarily slow down tests. Search how I've done that with fields that are collections on other types.

Reply:
I already have a limit in `RegistryEntry::arbitrary_with`. Is that sufficient?

https://github.com/flox/flox/pull/1837/files#diff-64beb671f7bc45cb731cff1c25feff42959d231905915bed808f79d64d2fa9c8R451